packages/theme-language-server-common/src/server/startServer.ts

```diff
@@ -35,11 +35,28 @@ import { GetTranslationsForURI } from '../translations';
 import { Dependencies } from '../types';
 import { debounce } from '../utils';
 import { VERSION } from '../version';
-import { Configuration } from './Configuration';
 import { CachedFileSystem } from './CachedFileSystem';
+import { Configuration } from './Configuration';
 
 const defaultLogger = () => {};
 
+/**
+ * The `git:` VFS does not support the `fs.readDirectory` call and makes most things break.
+ * `git` URIs are the ones you'd encounter when doing a git diff in VS Code. They're not
+ * real files, they're just a way to represent changes in a git repository. As such, I don't
+ * think we want to sync those in our document manager or try to offer document links, etc.
+ *
+ * A middleware would be nice but it'd be a bit of a pain to implement.
+ */
+const hasUnsupportedDocument = (params: any) => {
```

Comment:
Any chance we can type `params` here? Maybe the language server exports something?